web_development_101/project_html_css.md

```diff
@@ -94,6 +94,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 Submit a pull request with a link to your solution in this [file](https://github.com/TheOdinProject/curriculum/edit/master/web_development_101/project_html_css.md) when your project is complete. See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.
 
 * Add your solution below this list
+* [Dillon Gavlock's Solution (Easy)] (https://github.com/dgavlock/google-homepage) - [View in Browser] (https://dgavlock.github.io/google-homepage/)
```

Comment:
Notice how you have a space here? 
![image](https://user-images.githubusercontent.com/4411701/43846415-eb43baac-9af3-11e8-8347-394c8e312973.png)

To format correctly, you can remove those spaces, then we'll merge it :) You should be able to edit the commit, if you need help, feel free to respond and ask questions!

Good job on the page!